app/Model/TaskProjectDuplicationModel.php

```diff
@@ -33,6 +33,9 @@ public function duplicateToProject($task_id, $project_id, $swimlane_id = null, $
             $this->taskLinkModel->create($new_task_id, $task_id, 4);
         }
 
+        $hook_values = [ 'source_task_id' => $task_id, 'target_task_id' => $new_task_id];
+        $this->hook->reference('model:task:duplication:aftersave', $hook_values);
```

Comment:
This hook should be named differently that the first one? It's specific to project duplication.

Reply:
changed